packages/test-scenarios/__tests__/scenario-test.js

```diff
@@ -66,6 +72,47 @@ module.exports = function (defaults) {
   });
 }
 
+async function embroiderInRepoAddon(project) {
+  await embroider(project);
+  await addInRepoAddon(project, 'fake-addon');
+}
+
+async function addInRepoAddon(project, name, version = '0.0.0') {
+  project.linkDependency('ember-add-in-repo-tests', {
+    baseDir: __dirname,
+  });
+
+  project.pkg['ember-addon'] = {
+    paths: [`lib/${name}`],
```

Comment:
will we need to use join to generate these paths?

Reply:
We could, but our apps/tests are all assuming *nix right now. I know we discussed windows support, but we should consider adding that when needed.